nixos/modules/services/misc/freeswitch.nix

```diff
@@ -0,0 +1,102 @@
+{ config, lib, pkgs, ...}:
+with lib;
+let
+  cfg = config.services.freeswitch;
+  pkg = cfg.package;
+  configDirectory = pkgs.runCommand "freeswitch-config-d" { } ''
+    mkdir -p $out
+    cp -r ${cfg.configTemplate}/* $out
```

Comment:
Could use `cp -rT ${cfg.configTemplate} $out` which should copy dot-files too.

Reply:
Done.